test/models/mnist/mnist_deep_simplified.py

```diff
@@ -0,0 +1,258 @@
+        y_conv = tf.matmul(h_fc1, W_fc2) + b_fc2
+    return y_conv, tf.placeholder(tf.float32)
+
+
+def conv2d(x, W):
+    """conv2d returns a 2d convolution layer with full stride."""
+    return tf.nn.conv2d(x, W, strides=[1, 1, 1, 1], padding='SAME')
+
+
+def max_pool_2x2(x):
+    """max_pool_2x2 downsamples a feature map by 2X."""
+    return tf.nn.max_pool(
+        x, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding='SAME')
+
+
+def weight_variable(shape, name):
+    """weight_variable generates a weight variable of a given shape."""
+    weight_var = tf.get_variable(name, shape)
+    return weight_var
+
+
+def bias_variable(shape):
+    """bias_variable generates a bias variable of a given shape."""
+    initial = tf.constant(0.1, shape=shape)
+    return tf.Variable(initial)
+
+
+def train_mnist_cnn(FLAGS):
+    # Config
+    config = tf.ConfigProto(
+        allow_soft_placement=True,
+        log_device_placement=False,
+        inter_op_parallelism_threads=1)
+    # Enable the custom optimizer using the rewriter config options
+    #config = ngraph_bridge.update_config(config)
+
+    # Note: Additional configuration option to boost performance is to set the
+    # following environment for the run:
+    # OMP_NUM_THREADS=44 KMP_AFFINITY=granularity=fine,scatter
+    # The OMP_NUM_THREADS number should correspond to the number of
+    # cores in the system
+
+    # Import data
+    mnist = input_data.read_data_sets(FLAGS.data_dir, one_hot=True)
+
+    # Create the model
+    x = tf.placeholder(tf.float32, [None, 784])
+
+    # Define loss and optimizer
+    y_ = tf.placeholder(tf.float32, [None, 10])
+
+    # Build the graph for the deep net
+    y_conv, keep_prob = deepnn(x)
+
+    with tf.name_scope('loss'):
+        cross_entropy = tf.nn.softmax_cross_entropy_with_logits(
+            labels=y_, logits=y_conv)
+    cross_entropy = tf.reduce_mean(cross_entropy)
+
+    with tf.name_scope('adam_optimizer'):
+        train_step = tf.train.AdamOptimizer(1e-4).minimize(cross_entropy)
+
+    with tf.name_scope('accuracy'):
+        correct_prediction = tf.equal(tf.argmax(y_conv, 1), tf.argmax(y_, 1))
+        correct_prediction = tf.cast(correct_prediction, tf.float32)
+    accuracy = tf.reduce_mean(correct_prediction)
+    tf.summary.scalar('Training accuracy', accuracy)
+    tf.summary.scalar('Loss function', cross_entropy)
+
+    graph_location = "/tmp/" + getpass.getuser(
+    ) + "/tensorboard-logs/mnist-convnet"
+    print('Saving graph to: %s' % graph_location)
+
+    merged = tf.summary.merge_all()
+    train_writer = tf.summary.FileWriter(graph_location)
+    train_writer.add_graph(tf.get_default_graph())
+
+    saver = tf.train.Saver()
+
+    with tf.Session(config=config) as sess:
+        sess.run(tf.global_variables_initializer())
+        train_loops = FLAGS.train_loop_count
+        loss_values = []
+        for i in range(train_loops):
+            batch = mnist.train.next_batch(FLAGS.batch_size, shuffle=False)
```

Comment:
shuffle = False is set for the same reason

Reply:
If the only changes are setting the seed and shuffle = False, we can add these changes in the main file and not create a new file